### ISSUE TITLE
outputGenerator tweaks

### ISSUE THREAD

**Issue by missinglink** [pelias/api#46]
In the following example, the 'text' should be **'Arbil, Iraq'** but due to the way the `outputGenerator` is written it matches nothing for `local` and 'Arbil' for `regional` (the `name` is also 'Arbil').

If I simply add `admin1` to the end of the [local array](https://github.com/pelias/api/blob/suggest_plus_mget/helper/outputSchema.json#L11) then both `local` and `regional` will match 'Arbil' and when the de-duper removes one entry, we are back with the text being 'Arbil'.

Not sure if I described this well, but basically we need to continue checking values in the `local` and `regional` arrays until we find a value which is not a duplicate of the name or another admin value, otherwise a POI with ```{ "name": "a", "local_admin": "a", "admin1_abbr": "a", "admin0": "b" }``` will always just be `a` when it would be nicer if it were `a, b`.

Hope that makes sense ;)

```javascript
{
  "type": "FeatureCollection",
  "features": [
    {
      "type": "Feature",
      "geometry": {
        "type": "Point",
        "coordinates": [
          44.180874,
          36.369878
        ]
      },
      "properties": {
        "id": "1110:adm1:iq:irq:arbil",
        "type": "admin1",
        "layer": "admin1",
        "name": "Arbil",
        "alpha3": "IRQ",
        "admin0": "Iraq",
        "admin1": "Arbil",
        "text": "Arbil"
      }
    },
...
```

**Comment by hkrishna** [comment]
This bug will get addressed/fixed in #100

**Comment by dianashk** [comment]
Revisit after API redesign
